setup.py

```diff
@@ -14,7 +14,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/algorand/pyteal",
-    packages=["pyteal"],
+    packages=setuptools.find_packages(include=("pyteal*",)),
```

Comment:
@CiottiGiorgio Can you share how was the change tested?

I'm not immediately familiar with this part of Python's tooling ecosystem and I'm hoping you can accelerate my ability to validate the PR.

Reply:
Yes sorry I didn't describe appropriately how the issue was found and how it was solved (or what the issue even is).

So by explicitly setting `packages` to `["pyteal"]`, I excluded all submodules.
I found this out because I needed a specific commit for PyTeal after the latest release. I just pulled from master and I realized that no submodule was being packaged.
With the help of , I changed the inclusion rule for the packaging, deleted my venv, started fresh and reinstalled with `pip install .`.
This seems to have fixes this issue while still only importing `pyteal` root package + submodules.

Reply:
@CiottiGiorgio Resolving - Thanks for the explanation.  I reproduced the problem with these instructions.

Unfortunately, I didn't find a reasonable CI check to install during my several minute review.